genie-web/src/main/java/com/netflix/genie/web/controllers/JobRestController.java

```diff
@@ -227,27 +227,27 @@ public JobRestController(
     @RequestMapping(method = RequestMethod.POST, consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
     @ResponseStatus(HttpStatus.ACCEPTED)
     public ResponseEntity<Void> submitJob(
-        @RequestPart("request")
-        final JobRequest jobRequest,
-        @RequestPart("attachment")
-        final MultipartFile[] attachments,
-        @RequestHeader(value = FORWARDED_FOR_HEADER, required = false)
-        final String clientHost,
-        @RequestHeader(value = HttpHeaders.USER_AGENT, required = false)
-        final String userAgent,
-        final HttpServletRequest httpServletRequest
+            @RequestPart("request")
+            final JobRequest jobRequest,
+            @RequestPart("attachment")
+            final MultipartFile[] attachments,
+            @RequestHeader(value = FORWARDED_FOR_HEADER, required = false)
+            final String clientHost,
+            @RequestHeader(value = HttpHeaders.USER_AGENT, required = false)
+            final String userAgent,
+            final HttpServletRequest httpServletRequest
     ) throws GenieException {
         log.info("[submitJob] Called multipart method to submit job: {}", jobRequest);
         this.submitJobWithAttachmentsRate.increment();
         return this.handleSubmitJob(jobRequest, attachments, clientHost, userAgent, httpServletRequest);
     }
 
     private ResponseEntity<Void> handleSubmitJob(
-        final JobRequest jobRequest,
-        final MultipartFile[] attachments,
-        final String clientHost,
-        final String userAgent,
-        final HttpServletRequest httpServletRequest
+            final JobRequest jobRequest,
```

Comment:
It looks like your IDE is formatting things differently. Is it not picking up the .editorconfig file?